js/ui/editor/ui.data_expression.js

```diff
@@ -41,6 +42,7 @@ const DataExpressionMixin = extend({}, DataHelperMixin, {
                 store: new ArrayStore(this.option('items')),
                 pageSize: 0
             });
+            this._dataController = new DataController(this._dataSource);
```

Comment:
why do we create dataSource here manually, not use `initDataController` function?

Reply:
done